src/components/interlocutor/screens/InterlocutorList/actions/index.js

```diff
@@ -17,27 +17,48 @@ export function getInterlocutorList() {
 }
 
 export function getInterlocutorListFromRequest(request) {
-  if ( request.length > 0 ) {
-    return (dispatch, getState) => executeSql(
-      `SELECT name, link_to_me, id 
+  if (request.length > 0) {
+    requestList = request.split(' ');
+    return (dispatch, getState) => {
+      let promises = [];
+      request = `SELECT name, link_to_me, id 
       FROM interlocutor
-      WHERE name LIKE ? OR link_to_me LIKE ?`,
-      ["%" + request + "%", "%" + request + "%"]
-    )
-      .then((res) => {
-        const interlocutorListFromRequest = res.rows._array.map(interlocutor => ({
-          interlocutor: {
-            name: interlocutor.name,
-            link_to_me: interlocutor.link_to_me,
-            id: interlocutor.id,
-          }
-        }));
-        dispatch({ type: GET_INTERLOCUTOR_LIST_FROM_REQUEST, interlocutorListFromRequest });
-        return res
+      WHERE name LIKE ? OR link_to_me LIKE ?`;
+      requestList.forEach((term) => {
+        if (term.length > 0) {
+          promises = promises.concat(executeSql(request, [`%${  term  }%`, `%${  term  }%`]));
+        }
       });
+      return Promise.all(promises)
+        .then((res) => {
+          let interlocutorListFrequency = {}; // Key : interlocutor_id ; Value : [interlocutor object, frequency]
+          res.forEach((r) => {
+            r.rows._array.forEach((interlocutor) => {
+              if (interlocutor.id in interlocutorListFrequency) {
+                interlocutorListFrequency[interlocutor.id][1] += 1;
+              } else {
+                interlocutorListFrequency[interlocutor.id] = [{
+                  interlocutor: {
+                    name: interlocutor.name,
+                    link_to_me: interlocutor.link_to_me,
```

Comment:
Just noticing (not requesting change) : link_to_me is not following camelCase convention

Reply:
Ok not changed. but noticed